system-addon/content-src/components/Base/_Base.scss

```diff
@@ -93,19 +99,15 @@ main {
     background-color: transparent;
     border: 0;
     cursor: pointer;
-    fill: $fill-secondary;
+    fill: var(--newtab-icon-primary-color);
     offset-inline-end: 15px;
     padding: 15px;
     position: fixed;
     top: 15px;
     z-index: 12001;
 
     &:hover {
-      background-color: $background-secondary;
-    }
-
-    &:active {
-      background-color: $background-primary;
+      background-color: $grey-20;
```

Comment:
This should be grey-20 when clicking always?

Reply:
This is the hover on the prefs gear icon. For overall consistency, this should probably match `--newtab-section-active-color` so I should rename that.